include/boost/geometry/geometries/polyhedral_surface.hpp

```diff
@@ -0,0 +1,99 @@
+// Boost.Geometry
+
+// Copyright (c) 2025 Oracle and/or its affiliates.
+// Contributed and/or modified by Vissarion Fysikopoulos, on behalf of Oracle
+
+// Use, modification and distribution is subject to the Boost Software License,
+// Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+
+
+#ifndef BOOST_GEOMETRY_GEOMETRIES_POLYHEDRAL_SURFACE_HPP
+#define BOOST_GEOMETRY_GEOMETRIES_POLYHEDRAL_SURFACE_HPP
+
+#include <memory>
+#include <vector>
+#include <boost/concept/assert.hpp>
+#include <boost/geometry/geometries/concepts/point_concept.hpp>
+#include <boost/geometry/geometries/concepts/polygon_concept.hpp>
+#include <boost/geometry/core/tag.hpp>
+#include <boost/geometry/core/tags.hpp>
+#include <boost/config.hpp>
+
+namespace boost { namespace geometry
+{
+namespace model
+{
+
+/*!
+\brief A Polyhedral Surface is a contiguous collection of polygons,
+    which share common boundary segments.
```

Comment:
we don't check it. Suggestion: 
"The concepts and algorithms don't check if the boundary segments are common"

Reply:
I also added that common boundary segments should be checked by is_valid()